output/pursuits.json

```diff
@@ -0,0 +1,3692 @@
+  },
+  "50742572": {
+    "Place": [2426873752]
+  },
+  "53973468": {
+    "ActivityMode": [1418469392, 3894474826]
+  },
+  "53973469": {
+    "ActivityMode": [1418469392, 3894474826]
+  },
+  "53973470": {
+    "ActivityMode": [1418469392, 3894474826]
+  },
+  "53973471": {
+    "ActivityMode": [1418469392, 3894474826]
+  },
+  "56164121": {
+    "ActivityMode": [1848252830]
+  },
+  "67350175": {
+    "Place": [4251857532]
+  },
+  "81214651": {
+    "ActivityMode": [1164760504, 1826469369],
+    "DamageType": [3454344768],
+    "ItemCategory": [6]
+  },
+  "85374308": {
+    "ActivityMode": [1826469369]
+  },
+  "88202002": {
+    "Place": [3325508439]
+  },
+  "88348293": {
+    "ActivityMode": [1686739444]
+  },
+  "88669873": {
+    "ActivityMode": [1164760504],
+    "ItemCategory": [7]
+  },
+  "88669875": {
+    "ActivityMode": [1164760504]
+  },
+  "88669876": {
+    "ActivityMode": [1164760504],
+    "ItemCategory": [7]
+  },
+  "88669877": {
+    "ActivityMode": [1164760504],
+    "DamageType": [1847026933, 2303181850, 3454344768]
+  },
+  "88669878": {
+    "ActivityMode": [1164760504, 3199098480]
+  },
+  "88669879": {
+    "ActivityMode": [1164760504]
+  },
+  "90105468": {
+    "Place": [1259908504]
+  },
+  "99849256": {
+    "ActivityMode": [803838459],
+    "ItemCategory": [12]
+  },
+  "102952779": {
+    "ActivityMode": [803838459],
+    "ItemCategory": [5]
+  },
+  "107475132": {
+    "Place": [386951460]
+  },
+  "112356175": {
+    "Place": [4251857532]
+  },
+  "125824834": {
+    "DamageType": [2303181850]
+  },
+  "128980839": {
+    "Place": [2877881518]
+  },
+  "130487539": {
+    "ItemCategory": [153950757]
+  },
+  "131173969": {
+    "ActivityMode": [803838459]
+  },
+  "141625708": {
+    "Place": [3526908984]
+  },
+  "142905935": {
+    "ActivityMode": [803838459],
+    "Place": [3747705955]
+  },
+  "146278979": {
+    "DamageType": [2303181850]
+  },
+  "146645456": {
+    "Place": [3526908984]
+  },
+  "147797594": {
+    "Place": [3747705955]
+  },
+  "159003722": {
+    "ItemCategory": [3954685534]
+  },
+  "163419755": {
+    "Place": [3747705955]
+  },
+  "163476113": {
+    "Place": [3526908984]
+  },
+  "176941175": {
+    "Place": [3526908984]
+  },
+  "179306417": {
+    "ItemCategory": [13]
+  },
+  "180861015": {
+    "Place": [3747705955, 1259908504]
+  },
+  "188041967": {
+    "ActivityMode": [2394616003],
+    "Place": [975684424]
+  },
+  "192890964": {
+    "Place": [3747705955]
+  },
+  "196262240": {
+    "Place": [3526908984]
+  },
+  "196262241": {
+    "Place": [3526908984]
+  },
+  "196262242": {
+    "Place": [3526908984]
+  },
+  "196262246": {
+    "ActivityMode": [2394616003],
+    "Place": [3526908984]
+  },
+  "202169029": {
+    "Place": [3747705955, 386951460, 3526908984, 4251857532, 3325508439]
+  },
+  "205359284": {
+    "Place": [1259908504]
+  },
+  "206141942": {
+    "ActivityMode": [2394616003, 3789021730]
+  },
+  "206141943": {
+    "ActivityMode": [803838459]
+  },
```

Comment:
Also Escalation Protocol and Blind Well

Reply:
Unfortunately neither has an activity mode. Probably because they don't get PGCRs.